dbld/rules

```diff
@@ -59,6 +59,9 @@ bootstrap: bootstrap-$(DEFAULT_IMAGE)
 bootstrap-%: setup
 	$(DOCKER) run $(DOCKER_RUN_ARGS) --rm -e CONFIGURE_OPTS="$${CONFIGURE_OPTS:-$(CONFIGURE_OPTS)}" -ti  balabit/syslog-ng-$* /source/dbld/bootstrap
 
+make-%: setup
+	$(DOCKER) run $(DOCKER_RUN_ARGS) --rm -ti  balabit/syslog-ng-$(DEFAULT_IMAGE) /source/dbld/make $(MAKE_ARGS) $*
```

Comment:
I have two comments for this.
1, Is it enough to execute only a "make" on source? make install is not needed?
2, I can not find the definition of MAKE_ARGS (or is it a general env. variable)

Reply:
you can call the makefile using:

```
$ dbld/rules make-install MAKE_ARGS=-j4
```

make works so that if a variable is not defined it is populated from the environment or the command line.

The above rule does not support make without any arguments, but the default target for make is usually "all", so you can achive the same with:

```
$ dbld/rules make-all
```

The whole thing is about being able to compile right from the source directory, without having to enter the docker container using "shell".